EventGenerator/src/PionFilter_module.cc

```diff
@@ -61,23 +66,30 @@ namespace mu2e {
         for(const auto& aParticle : *ah){
           art::Ptr<SimParticle> pp(ah, aParticle.first.asUint());
           float _endglobaltime = pp->endGlobalTime();
-          if( pp->stoppingCode() == ProcessCode::mu2eKillerVolume and std::abs(pp->pdgId()) == PDGCode::pi_plus){
+          if( pp->stoppingCode() == ProcessCode::hBertiniCaptureAtRest and std::abs(pp->pdgId()) == PDGCode::pi_plus){
```

Comment:
Is this selection dependent on physics list? Ie ci
Outdoor the pion stop process change? If so this should be a configurable parameter.

Reply:
I dont think it should be